### DISCUSSION TITLE
temporal slider

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [vasturiano/3d-force-graph#685] [Ideas]
I would like to use temporal data with time slider, would be even better if it was 3d ;)

https://observablehq.com/@d3/temporal-force-directed-graph

**Comment by omfgroflmfaol** [comment]
Have you been able to implement this?

**Reply by f1-outsourcing** [reply]
yes more or less

**Reply by omfgroflmfaol** [reply]
Great to hear that!
I also need a "temporal data filtering" functionality for my graph, but I am completely new to js so I wouldn't know where to start in order to implement it. If you could share your code, I would much appreciate it :) It doesn't need to be pretty, I can ask ChatGPT to explain it to me... but it would be great to see an implementation that works!
